Documentation/GeneralConventions/Glossary.rst

```diff
@@ -119,6 +119,11 @@ O
 P
 
 `page TSconfig`,
+`PhpStorm`,
+`PHPStan`,
+`PHPUnit`,
+`PHP CS Fixer`,
+`PHP_CodeSniffer`,
```

Comment:
With underscore? OK...in about-section of their GitHub repo it was also written that way. Ugly. Brrr. Approved.

Reply:
Yes in their Wiki and everywhere they utilize this spelling. Well... some people also thing TYPO3 is odd :D